config/settings.defaults.coffee

```diff
@@ -52,6 +52,11 @@ settings =
 	
 	max_doc_length: 2 * 1024 * 1024 # 2mb
 
+	# combine
+	# max_doc_length (2mb see above) * 2 (delete + insert)
+	# max_ranges_size (3mb see MAX_RANGES_SIZE in doc-updater) 
+	maxUpdateSize: parseInt(process.env['MAX_UPDATE_SIZE']) or 7 * 1024 * 1024
```

Comment:
is 7Mb from 3mb * 2 + some headroom?

Reply:
It's 3mb (ranges) + 2 * 2mb (doc lines). I could not max out both doc lines and ranges. Anyways I will add some headroom.